README.md

```diff
@@ -25,23 +25,21 @@ import (
 
 ### Go Module Support
 
-The library supports Go's [experimental modules][modules]. Add stripe-go as a
-requirement in your `go.mod` along with its current version:
+The library currently *does not* ship with first-class support for Stripe
+modules. We put in support for it before, but ran into compatibility problems
+for existing installations using Dep (see discussion in [closer to the bottom
+of this thread][gomodvsdep], and [reverted support][gomodrevert]. Our current
+plan is to wait for better module compatibility in Dep (see a [preliminary
+patch here][depgomodsuppor]), give the release a little grace time to become
+more widely distributed, then bring support back.
+
+For now, require stripe-go in `go.mod` like this:
 
 ``` go
 module github.com/my/package
 
 require (
-    github.com/stripe/stripe-go/v55 v55.9.0
-)
-```
-
-Imports should contain the major version as a virtual component of each path:
-
-``` go
-import (
-    "github.com/stripe/stripe-go/v55"
-    "github.com/stripe/stripe-go/v55/customer"
+    github.com/stripe/stripe-go/v55
```

Comment:
I don't know if this is right. The modules Wiki [has some information on this](https://github.com/golang/go/wiki/Modules#can-a-module-consume-a-v2-package-that-has-not-opted-into-modules-what-does-incompatible-mean), but naturally provides no examples. Let me get #774 for in and just try it out to make sure I get the right invocation.